### ISSUE TITLE
Default value of transmission_dispersion_abbe_number is not between uisoftmin and uisoftmax

### ISSUE THREAD

**Issue by JGamache-autodesk** [AcademySoftwareFoundation/OpenPBR#200]
The soft min and max of that parameter are between 9.0 and 91.0, yet the default value is 0.0. I would have expected the default value to sit between the soft min and max (for UI purpose).

**Comment by jstone-lucasfilm** [comment]
Good catch, @JGamache-autodesk, and I believe this is simply a bug that should be addressed for v1.0.

I'm CC'ing @peterkutz and @portsmouth for confirmation, but I believe the intended default for this parameter is 20.0.

**Comment by portsmouth** [comment]
Yes it should be 20.

**Comment by portsmouth** [comment]
@JGamache-autodesk can this be closed?
